suzieq/cli/sqcmds/command.py

```diff
@@ -291,41 +326,57 @@ def _do_help(self, table: str, verb: str = 'show'):
             verb (str, optional): the name of the command. Defaults to 'show'.
         """
 
-        mbrs = inspect.getmembers(self)
-        classargspec = [x[1] for x in mbrs
-                        if x[0] == '__arguments_decorator_specs']
-        if classargspec:
-            classargspec = classargspec[0]
-        verbs = [x[0] for x in mbrs
-                 if inspect.ismethod(x[1]) and not x[0].startswith('_')]
-        fnlist = {x[0]: x[1] for x in mbrs
-                  if x[0] in verbs}
-        newverb = verb.replace('assert', 'aver')
-        if newverb in fnlist:
-            fnargs = []
-
-            fnmbrs = inspect.getmembers(fnlist[newverb])
-            for elem in fnmbrs:
-                if elem[0] == "__arguments_decorator_specs":
-                    fnargs = elem[1]
-                    break
-
-            docstr = inspect.getdoc(fnlist[newverb])
-            if docstr:
-                docstr = docstr.splitlines()[0]
-            else:
-                docstr = ''
-            print(f"{table} {verb}: " + Fore.CYAN +
-                  f"{docstr}" + Style.RESET_ALL)
-            print(Fore.YELLOW + '\nArguments:' + Style.RESET_ALL)
-            if fnargs:
-                classargspec.update(fnargs)
-            classargspec.update(self._additional_help_vars)
-            for arg in sorted(classargspec):
-                print(f" - {arg}: " + Fore.CYAN +
-                      f"{classargspec[arg].description}" + Style.RESET_ALL)
-        else:
+        # Determine if the method is present
+        if verb == 'assert':
+            verb = 'aver'
```

Comment:
What happens here, why does it get changed to 'aver'?

Reply:
Becaise the function is called aver, not assert. assert is a reserved keyword in python that cannot be used as a function name.